pages/search.js

```diff
@@ -0,0 +1,32 @@
+import { useEffect, useState } from 'react';
+import { useRouter } from 'next/router';
+import ArticleList from '@/components/ArticleList';
+import SearchForm from '@/components/SearchForm';
+import axios from '@/lib/axios';
+import styles from '@/styles/Search.module.css';
+
+export default function Search() {
+  const [article, setArticle] = useState([]);
```

Comment:
P2:
타입을 명시해주시면 좋을 것 같아요.
이름도 배열임을 알 수 있게 해주시면 더 좋겠습니다~

```suggestion
  const [article, setArticle] = useState<ArticleProps[]>([]);
```